lib/pengine/pe_output.c

```diff
@@ -2714,9 +2716,15 @@ resource_list(pcmk__output_t *out, va_list args)
             printed_header = true;
         }
 
+        // User-supplied description
+        if (pcmk_is_set(show_opts, pcmk_show_rsc_only)
+            || pcmk__list_of_multiple(rsc->running_on)) {
+            desc = crm_element_value(rsc->xml, XML_ATTR_DESC);
+        }
+
         /* Print this resource */
         x = out->message(out, crm_map_element_name(rsc->xml), show_opts, rsc,
-                         only_node, only_rsc);
+                         only_node, only_rsc, desc);
```

Comment:
FYI, crm_map_element_name() gets the XML element type used for the resource, so it will be "primitive", "group", "clone", or "bundle".

fmt_functions[] in lib/pengine/pe_output.c maps those strings to actual functions depending on the output type. For example, "clone" calls either pe__clone_default() or pe__clone_xml().

So, updating the messages will involve updating all those functions.

Reply:
pe__bundle_xml pe__bundle_text pe__resource_text pe__resource_xmlpe__resource_html pe__bundle_html